packages/styled-components/src/test/types.tsx

```diff
@@ -38,7 +38,18 @@ const Component2 = () => {
   return (
     // Wrapper should not be influenced by Button's types
     <Wrapper>
-      <Button as="video" foo loop>
+      <Button as={'video'} foo loop>
+        test
+      </Button>
+    </Wrapper>
+  );
+};
+
+const Component3 = () => {
+  return (
+    // Wrapper should not be influenced by Button's types
+    <Wrapper>
+      <Button mix={['video' as const]} foo loop>
```

Comment:
For some reason typescript do not treat 'video' as const string

Reply:
To have const string you can also mark entire array as const otherwise it will be `string[]` by default

Reply:
I think I should just remove strings in `mix` since there is no sense to mix plain tags.